README.md

```diff
@@ -1367,6 +1369,90 @@ const info = await ant.getInfo();
 
 </details>
 
+### `getHandlers()`
+
+Retrieves the handlers supported on the ANT
+
+```typescript
+const handlers = await ant.getHandlers();
+```
+
+<details>
+  <summary>Output</summary>
+
+```json
+[
+  "evolve",
+  "_eval",
+  "_default",
+  "transfer",
+  "balance",
+  "balances",
+  "totalSupply",
+  "info",
+  "addController",
+  "removeController",
+  "controllers",
+  "setRecord",
+  "removeRecord",
+  "record",
+  "records",
+  "setName",
+  "setTicker",
+  "initializeState",
+  "state"
+]
+```
+
+</details>
+
+### `validate({ validations })`
+
+Validate the APIs exposed by the ANT. Defaults to verifying the requisite handlers are returned.
+
+```typescript
+const validations = {
+  ["setTicker"] = async ({ant}) => {
+      const handlers = await ant.getHandlers()
```

Comment:
if this `validate` function is internal to the ANT class, wouldn't this be this.getHandlers(). and is this example aligned with the output below?